content/download/lts/index.adoc

```diff
@@ -92,7 +92,8 @@ See the link:/content/event-calendar[event calendar] for the specific LTS RC/rel
 
 ## Backporting Process
 
-Any user can propose that a bug fix be backported to LTS by labeling with link:https://issues.jenkins.io/secure/IssueNavigator.jspa?reset=true&jqlQuery=labels+%3D+lts-candidate[lts-candidate].
+Any user can propose that a issue is backported to LTS by labeling with link:https://issues.jenkins.io/secure/IssueNavigator.jspa?reset=true&jqlQuery=labels+%3D+lts-candidate[lts-candidate].
+For an issue to be eligable for backporting it should be a small, safe-looking, uncontroversial fix for an important bug with no change to API surface, additionally library updates can also be considered if the library contains a public CVE that does not affect Jenkins (if Jenkins is vulnerable via a library then the link:/content/security[security] process applies).
```

Comment:
```suggestion
For an issue to be eligible for backporting it should be a small, safe-looking, uncontroversial fix for an important bug with no change to API surface, additionally library updates can also be considered if the library contains a public CVE that does not affect Jenkins (if Jenkins is vulnerable via a library then the link:/content/security[security] process applies).
```

Reply:
```suggestion
For an issue to be eligable for backporting it should be a small, safe-looking, uncontroversial fix for an important bug with no change to API surface.
Additionally library updates can also be considered if the library contains a public CVE that does not affect Jenkins (if Jenkins is vulnerable via a library then the link:/content/security[security] process applies).
```

Reply:
Would we not consider backports of library upgrades without public CVEs? Is that the important difference here?

Reply:
Yes,
Currebtly if Jenkins had a bug that was caused by a library it would be already eligible if upgrading the library fixed the bug.
If the library update is just to shut up a scanner it would not be eligable per my understanding as it does not fix a Jenkins bug.

Reply:
We backported https://issues.jenkins.io/browse/JENKINS-64035 🤷 

TBH I think we don't need an update of this documentation to accomplish your goal (in particular because this is more user documentation than developer documentation). I expect you'll just need to file issues suggesting backports that appear in the query results. If that doesn't work, we can always reconsider.

Reply:
Certainly internally within my team we where confused and did not think that this backports like this be acceptable.  Some of us are core committers and others are not (so more akin to users)

Reply:
Any change can be proposed for backporting 🤷  This documentation IMO also aims to reduce the number of new features requested to be backported by people who only use LTS for stability, but want that great new feature RIGHT NOW.

Reply:
(Weird, I tried adding separate comments for the same line, but it didn't work.)